### ISSUE TITLE
An option to force the alphabetic order of folders

### ISSUE THREAD

**Issue by nekrostr** [Betterbird/thunderbird-patches#354]
There seems to be a default alphabetical order of folders in the folder pane which is overridden if the user drags and drops a folder between two folders. This action makes the user prone to inadvertently set a folder on manual sort somehow as it doesn't abide by the alphabetical sort anymore. Please add an option, or an advanced config setting, somewhere to force the alphabetical order everywhere and disable the manual override.

Also If a folder is set in manual sort this way, there doesn't seem to be an option to reset it, a context menu perhaps in the parent folder to sort the children alphabetically, or to at least reset its individual 'manual' status somehow to follow the default order.

Thanks for the attention, and the hard work!

**Comment by Betterbird** [comment]
Yes, you observed correctly that once you move a folder in between its siblings, the automatic alphabetical order is lost. However, that shouldn't be too noticeable since new folders are still inserted in semi-alphabetical order and you can always re-establish the alphabetical order manually.

That said, there is no indication of which rules are applied and there is no way to reset the ordering scheme.

Would you like to fund this enhancement request?

**Comment by Betterbird** [comment]
The TB folks implemented that in TB 142, so we'll see whether that can be backported to BB 140.

**Comment by Betterbird** [comment]
So we included https://bugzilla.mozilla.org/show_bug.cgi?id=1972710 into BB 140. This will ship in BB 140.2.0 the week after next (19 Aug 2025). There is still a remaining issue, https://bugzilla.mozilla.org/show_bug.cgi?id=1979880, but we'll go with what we have for now.
